app/views/project/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<% content_for :pre_content_nav do %>
+<%= link_to "Back", root_path, class: "govuk-back-link" %>
+<% end %>
+
+<h1 class="govuk-heading-l">Add a new project</h1>
+
+<%= form_with model: @project, url: project_index_path do |form| %>
+  <%= form.govuk_text_field :urn, placeholder: "134679", label: { text: "What is the existing school's URN?", tag: 'h3', size: 'm' } %>
```

Comment:
It might be a me thing, but I am not convinced that the placeholder is valuable here? I would much rather see a hint:

https://design-system.service.gov.uk/components/text-input/